Sources/UIKit+Hero.swift

```diff
@@ -117,6 +117,46 @@ internal extension NSObject{
 }
 
 public extension UIViewController{
+  private struct PreviousDelegates {
+    static var navigationDelegate: UINavigationControllerDelegate? = nil
```

Comment:
You are correct! I don't know what I was thinking. I'll post a PR for this right away.